README.md

```diff
@@ -2,45 +2,154 @@
+
+```cmd
+> .\vcpkg\vcpkg search azure-
+```
+
+Once the library is installed, follow the instructions from the console output to include the library in your `CMake` application. For example, to include `azure-storage-blobs-cpp`, add the following to your `CMakeLists.txt` file:
+
+```CMake
+find_package(azure-storage-blobs-cpp CONFIG REQUIRED)
+target_link_libraries(myProject PRIVATE Azure::azure-storage-blobs)
+```
+
+> NOTE: All the Azure client libraries take a dependency on `azure-core-cpp` which provides functionality commonly needed by all Azure clients. When you install any client library via vcpkg, it will bring in all the necessary dependencies as well. You don't need to install those individually to get started.
+
+You can reference this [vcpkg Quick Start](https://github.com/microsoft/vcpkg#quick-start-windows) for more details.
+
+### Building your Application
+
+In order to use the SDK installed via vcpkg with CMake, you can use the toolchain file from vcpkg:
 
-### Client: New Releases
+```cmd
+> cmake -B [build directory] -S . -DCMAKE_TOOLCHAIN_FILE=[path to vcpkg repo]/scripts/buildsystems/vcpkg.cmake
+> cmake --build [build directory]
+```
 
-New wave of packages that we are announcing as **GA** and several that are currently releasing in **beta**. These libraries follow the [Azure SDK Design Guidelines for C++](https://azure.github.io/azure-sdk/cpp_introduction.html) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features at [Azure::Core](https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/core/azure-core/README.md).
+The **entry point** for most scenarios when using the SDK will be a top-level client type corresponding to the Azure service you want to talk to. For example, sending requests to blob storage can be done via the `Azure::Storage::Blobs::BlobClient` API.
 
-These new client libraries can be identified by the naming used for their folder, package, and namespace. Each will start with `azure`, followed by the service category, and then the name of the service. For example `azure-storage-blobs`.
+All the Azure C++ SDK headers needed to be included are located within the `<azure>` folder, with sub-folders corresponding to each service. Similarly, all types and APIs can be found within the `Azure::` namespace. For example, to use functionality form `Azure::Core`, include the following header at the beginning of your application `#include <azure/core.hpp>`.
+
+Here's an example application to help you get started:
+
+```C++
+#include <iostream>
+#include <azure/core.hpp>
+#include <azure/storage/blobs.hpp>
+
+using namespace Azure::Storage::Blobs;
+
+std::string GetConnectionString()
+{
+  return "get connection string from azure portal/environment";
+}

```

Comment:
Well, this gives the impression that we are recommending having the connection string in the source code, which is bad.

Since this is an example of how we would do a sample app, can we use `std::getenv()` to recommend always putting connection string on the env?

Reply:
This isn't really the point of the sample, and I didn't want to encourage use of `getenv()` especially without "is string empty" checks. Doing it properly ends up taking up too much code space. But you bring up a good point.

Maybe I should just leave it without a body and add a comment that the user can read it from an environment variable.